### PR TITLE
Add account address pub key to fms chain config for Starknet

### PR DESCRIPTION
**Pull request by eutopian**
Starknet needs the pub key in the job spec, so we are sending it down to CLO via the chain config.

### CODE REVIEW

File: core/services/feeds/orm_test.go [hunk 1]
```diff
@@ -206,11 +206,12 @@ func Test_ORM_CreateChainConfig(t *testing.T) {
...
-			ChainID:        "1",
-			ChainType:      feeds.ChainTypeEVM,
-			AccountAddress: "0x0001",
-			AdminAddress:   "0x1001",
+			FeedsManagerID:          fmID,
+			ChainID:                 "1",
+			ChainType:               feeds.ChainTypeEVM,
+			AccountAddress:          "0x0001",
+			AdminAddress:            "0x1001",
+			AccountAddressPublicKey: null.StringFrom("123"),
```
Comment (augustbleeds) [inline core/services/feeds/orm_test.go]: the public key would also be in hex format ("0x...."). might be helpful to clarify that by having "0x0100" as an example.

File: core/services/feeds/service_test.go [hunk 2]
```diff
@@ -489,14 +491,15 @@ func Test_Service_UpdateChainConfig(t *testing.T) {
...
-			AccountAddress:    "0x0000000000000000000000000000000000000000",
-			AdminAddress:      "0x0000000000000000000000000000000000000001",
-			FluxMonitorConfig: feeds.FluxMonitorConfig{Enabled: false},
-			OCR1Config:        feeds.OCR1Config{Enabled: false},
-			OCR2Config:        feeds.OCR2ConfigModel{Enabled: false},
+			FeedsManagerID:          mgr.ID,
+			ChainID:                 "42",
+			ChainType:               feeds.ChainTypeEVM,
+			AccountAddress:          "0x0000000000000000000000000000000000000000",
+			AccountAddressPublicKey: null.StringFrom("4ef1e379910e567ac08cb3fee297595ae41aba618eb7257ecee5e6e53a35f9f4"),
```
Comment (augustbleeds) [inline core/services/feeds/service_test.go]: yeah, let's prefix the hex value with 0x here as well for uniformity

File: core/web/resolver/feeds_manager_chain_config_test.go [hunk 3]
```diff
@@ -15,14 +15,15 @@ import (
...
-		accountAddr   = "0x0000001"
-		adminAddr     = "0x0000002"
-		forwarderAddr = "0x0000003"
-		peerID        = null.StringFrom("p2p_12D3KooWMoejJznyDuEk5aX6GvbjaG12UzeornPCBNzMRqdwrFJw")
-		keyBundleID   = null.StringFrom("6fdb8235e16e099de91df7ef8a8088e9deea0ed6ae106b133e5d985a8a9e1562")
+		mgrID          = int64(100)
+		cfgID          = int64(1)
+		chainID        = "42"
+		accountAddr    = "0x0000001"
+		acctAddrPubKey = "123"
```
Comment (augustbleeds) [inline core/web/resolver/feeds_manager_chain_config_test.go]: here too 👍

**Review by augustbleeds — DISMISSED** [review]
just some minor comments about making the test values hex

**Review by augustbleeds — DISMISSED** [review]

File: core/web/resolver/feeds_manager_chain_config.go [hunk 4]
```diff
@@ -43,6 +43,11 @@ func (r *FeedsManagerChainConfigResolver) AccountAddr() string {
 	return r.cfg.AccountAddress
 }
 
+// AccountAddr resolves the chain configs's account address.
```
Comment (chris-de-leon-cll) [inline core/web/resolver/feeds_manager_chain_config.go]:
```suggestion
// AccountAddrPubKey resolves the chain configs's account address public key.
```

**Review by chris-de-leon-cll — DISMISSED** [review]

**Review by chris-de-leon-cll — APPROVED** [review]

**Review by augustbleeds — APPROVED** [review]

**Pull request merged**